src/catkin_pkg/package.py

```diff
@@ -215,7 +215,7 @@ def validate(self, warnings=None):
             errors.append('Package description must not be empty')
 
         if not self.maintainers:
-            errors.append('Package must declare at least one maintainer')
+            errors.append('Package {} must declare at least one maintainer'.format(self.name))
```

Comment:
Please put `'` single quotes around the package name. It's something we try to do anytime we template in a variable, so it's more obvious if it ends up being empty or something.